grails-app/domain/org/pih/warehouse/inventory/TransactionSource.groovy

```diff
@@ -40,6 +40,13 @@ class TransactionSource {
     User createdBy
     User updatedBy
 
+    /**
+     A flag to indicate whether a transaction source was created during migration of existing transactions
+     The flag is to indicate whether the transaction source is "accurate" and is supposed to be showing truth for 100%
+     If migrated = true, it means there is a small risk of the source being not accurate
+    */
+    Boolean migrated
```

Comment:
That property can be misleading in the future, because someone who will be working on it in the future can have no idea what that means, because we can have a tons of migrations in the future.  I would add at least a URL to confluence / ticket.